Mixin/UserInterface/Controllers/Home/Apps/Cells/AppCell.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+
+class AppCell: UICollectionViewCell {
+    
+    @IBOutlet weak var imageView: AvatarImageView?
+    @IBOutlet weak var label: UILabel?
+    @IBOutlet weak var imageContainerView: UIView!
+    
+    var item: AppItem? {
+        didSet {
+            updateUI()
+        }
+    }
+    var snapshotView: HomeAppsSnapshotView {
+        let iconView = imageContainerView.snapshotView(afterScreenUpdates: true)!
```

Comment:
Do not force unwrap the return value. `snapshotViewAfterScreenUpdates:` doesn't guarantee a non-nil view is returned.